baby-gru/src/components/validation-tools/MoorhenIrisValidation.tsx

```diff
@@ -28,31 +29,71 @@ export const MoorhenIrisValidation = (props: {
     const width = useSelector((state: moorhen.State) => state.sceneSettings.width)
     const height = useSelector((state: moorhen.State) => state.sceneSettings.height)
     const molecules = useSelector((state: moorhen.State) => state.molecules)
+    const maps = useSelector((state: moorhen.State) => state.maps)
+
+    const [result, setResult] = useState();
 
     const random_data = generate_random_data(5) // get 5 metric rings
+    const [selectedModel, setSelectedModel] = useState<null | number>(null)
+    const [selectedMap, setSelectedMap] = useState<null | number>(null)
+    const mapSelectRef = useRef<undefined | HTMLSelectElement>();
+    const moleculeSelectRef = useRef<undefined | HTMLSelectElement>();
+
+    const handleModelChange = (evt: React.ChangeEvent<HTMLSelectElement>) => {
+        setSelectedModel(parseInt(evt.target.value))
+    }
+
+    const handleMapChange = (evt: React.ChangeEvent<HTMLSelectElement>) => {
+        setSelectedMap(parseInt(evt.target.value))
+    }
 
     const aes: IrisAesthetics = {
         dimensions: [plotDimensions, plotDimensions],
-        radius_change: 50, 
+        radius_change: 50,
         header: 40,
-        text_size: 50
+        text_size: 100
     }
 
     const results: IrisData = {
-        data: random_data,
-        chain_list: ["A", "B", "C"],
-        file_list: ["input1"], 
-    } 
-    
-    const iris_props: IrisProps = { 
+        data: result,
+        chain_list: null,
```

Comment:
Is the idea that the result data will already contain data for a specific chain and therefore `chain_list: null`?

Reply:
Yes the result of the backend_call contains the chain label information, I should probably make this an optional prop